api/.travis.env

```diff
@@ -16,7 +20,7 @@ DB_USERNAME=postgres
 DB_PASSWORD=
 
 CACHE_DRIVER=redis
-# SESSION_DRIVER=memcached
+SESSION_DRIVER=array
```

Comment:
@xiphiaz Just a quick follow up on this. Just to make sure the session doesn't get hit during auth, I profiled the route.

https://blackfire.io/profiles/4e36d1c2-5c92-4d17-909a-3f4bf5e1f790/graph

And it does not get hit, though having sessions in the system at all, do add a slight overhead. Session related operations are done at 3 occasions, regarding registration of provider, of manager and finally driver.

Roughly 1ms each, so a total of 3ms, so not very heavy. Though still, maybe worth keeping in mind, that if removing sessions completely, there is some extra cycles to be gained.